saleor/app/management/commands/create_app.py

```diff
@@ -25,51 +24,34 @@ def add_arguments(self, parser: CommandParser) -> None:
             help="Assign new permission to app."
             "Argument can be specified multiple times.",
         )
-        parser.add_argument("--is_active", default=True, dest="is_active")
         parser.add_argument(
-            "--target_url",
+            "--activate-after-installation", action="store_true", dest="activate"
```

Comment:
Can't be just `--activate`? And a description "Activates the app after installation". It's CLI so the less typing the easier to use for devs.